ocaml-lsp-server/test/e2e-new/documentation.ml

```diff
@@ -18,7 +18,7 @@ module Util = struct
   ;;
 
   let print_documentation result =
-    result |> Yojson.Safe.pretty_to_string ~std:false |> print_endline
```

Comment:
I think, instead of using `print_documentation result` you can just use `Test.print_result` in every test